entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+if [[ -z "$*" ]]; then
+  echo "Expecting one or more arguments"
+  exit 1
+fi
+
+#remove spack if exist previously
+if [[   -d "$GITHUB_WORKSPACE"/temp/spack ]]; then
+    rm -rf "$GITHUB_WORKSPACE"/temp
+fi
+
+# install Spack
+git clone https://github.com/spack/spack  "$GITHUB_WORKSPACE"/temp/spack&& \
```

Comment:
we can remove the `&& \` at the end of all these commands. This as a way of making it clear which from all these four commands fails, otherwise, when a failure occurs, it might be harder to identify which one did.